content/200-concepts/050-overview/100-what-is-prisma/100-data-modeling.mdx

```diff
@@ -61,6 +61,24 @@ It has the following columns:
 - `email`: A string with at most 255 characters. Additionally, the added constraints express that no two records can have duplicate values for the `email` column, and that _every_ record needs to have a value for it.
 - `isAdmin`: A boolean that indicates whether the user has admin rights (default value: `false`)
 
+#### MongoDB
+
+In MongoDB databases, models are represented by _collections_ and contain _documents_ that can have any structure:
+
+```js
+{
+  id: '607ee94800bbe41f001fd568',
```

Comment:
Is `id` actually possible or does it have to be `_id`?

Reply:
Good catch